src/devices/machine/ncr5385.cpp

```diff
@@ -759,9 +767,29 @@ int ncr5385_device::state_step()
 			else
 				m_sbx = false;
 
+#if 0
 			// clear data and ACK
 			scsi_bus->data_w(scsi_refid, 0);
 			scsi_bus->ctrl_w(scsi_refid, 0, S_ACK);
+#else
+			// clear ACK except after last byte of message output phase
```

Comment:
I don't think this is correct; this logic only applies for the message input phase, to allow the initiator the chance to raise ATN to reject a message from the target if necessary. The same thing does not apply during message output.